tsdb/tsm1/file_store.go

```diff
@@ -376,6 +376,14 @@ func (t *fileTracker) ClearFileCounts() {
 	}
 }
 
+func (t *fileTracker) ClearDiskSizes() {
+	labels := t.Labels()
+	for i := uint64(0); i <= 4; i++ {
```

Comment:
Doesn't this need to start at `1`?

Reply:
Yes sure, sorry the I did not see the '0' wrapped into the type :(